### ISSUE TITLE
Create new LocalizedViewThemer to support different langages

### ISSUE THREAD

**Issue by lonnieezell** [ci-bonfire/Sprint#70]
The purposed of creating a new Themer here is to keep performance and simplicity around for people who do NOT need the additional localization support. 
- Copy the existing ViewThemer to a new LocalizedViewThemer
- When displaying theme folders, should look for a subfolder in the theme folder whose name matches the value of LANGUAGE constant. If it exists, that file should be used in place of the stock one. If that file does not exist, it would display the default one in the themes folder. 
- The themes can override controller/method settings. These should also be looked for within the themes/{language} folder.
- When the `display` method is displaying a non-themed file, it should look in views/{language}/{controller}/{method}.php first. If it doesn't exist, then try to display a normal, non-localized version.

**Comment by IrcDirk** [comment]
Ok got one problem in part of LocalizedViewThemer:

```
// Otherwise, treat it as a standard view, which means
// application/views will override any modules. (See HMVC/Loader)
else
{
     $output = $this->ci->load->view( $view, $data, TRUE );
}
```

couse we use HMVC loader of views, so if i change: 

```
$output = $this->ci->load->view( $view, $data, TRUE );
```

to

```
if(defined('CURRENT_LANGUAGE')) $view = CURRENT_LANGUAGE.'/'.$view;
$output = $this->ci->load->view( $view, $data, TRUE );
```

It will throw error if `'en/'` in url couse it will not find ie `'english/home/index.php'`, but i cant add `file_exists` to `if`, it will not work here.

Maybe override HMVC function `views()` in `MY_Loader.php`?

Or maybe u got other sollution for this?

**Comment by lonnieezell** [comment]
Not sure I understand all of the situation. Can you submit a pull request with the current state and I'll take a look?

**Comment by IrcDirk** [comment]
Ok, new PR https://github.com/ci-bonfire/Sprint/pull/84

but whats is needed, is to support `application/views` and `modules/views` since they both using the same `if`: 
https://github.com/IrcDirk/Sprint/blob/90c06d91d180530d5ae7217c3c56ea1f9ebeba25/myth/Themers/ViewThemer.php#L586 
and native CI view function.

**Comment by lonnieezell** [comment]
Fixed with Pull Request #86
